### PR TITLE
net-misc/pcapfix: version bump 1.1.4, EAPI 7

### PR DESCRIPTION
**Pull request by arthurzam**
Bump version to 1.1.4
update ebuild to `EAPI=7`
add USE flag `debug`

Closes: https://bugs.gentoo.org/498752
Package-Manager: Portage-2.3.62, Repoman-2.3.12

### CODE REVIEW

File: net-misc/pcapfix/pcapfix-1.1.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+HOMEPAGE="http://f00l.de/pcapfix/"
+SRC_URI="http://f00l.de/pcapfix/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
```
Comment (juippis) [inline net-misc/pcapfix/pcapfix-1.1.4.ebuild]: This is rather useless :) You can leave them empty or even remove these variables if nothing goes into them.

File: net-misc/pcapfix/pcapfix-1.1.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+DOCS=( Changelog README )
+
+src_prepare() {
+	default
+	sed -e 's/gcc/$(CC) $(CFLAGS)/g' -i Makefile
```
Comment (juippis) [inline net-misc/pcapfix/pcapfix-1.1.4.ebuild]: Please append with `|| die`, you can have an informational message if you wish, too.

File: net-misc/pcapfix/pcapfix-1.1.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	use debug || sed -e 's/DEBUGFLAGS = -g/DEBUGFLAGS =/g' -i Makefile
```
Comment (juippis) [inline net-misc/pcapfix/pcapfix-1.1.4.ebuild]: This also needs to be appended with `|| die`. With EAPI-7, `(use debug || sed -e 's/DEBUGFLAGS = -g/DEBUGFLAGS =/g' -i Makefile) || die` should work, but if it doesn't, use `if use debug ; then` form instead.

**Comment by arthurzam** [comment]
@juippis Thank you
Fixed all.
Didn't know you can just remove the RDEPEND and DEPEND if they are empty.

**Comment by robbat2** [comment]
Merged, thanks

**Pull request closed without merging**